cogs/gamerwords.py

```diff
@@ -122,9 +215,30 @@ async def on_message_edit(self, old_message, new_message):
 		await self.handle_new_gamer_message(new_message)
 
 	async def handle_new_gamer_message(self, message):
-		match = self.has_gamer_words(message.content)
-		if match:
+		text_match = self.has_gamer_words(message.content)
+		file_match = any(
+			[self.has_gamer_words(attachment.filename) for attachment in message.attachments]
+		)
+		if text_match or file_match:
+			files = []
+			for attachment in message.attachments:
+				# 8MB
+				if attachment.size >= 8_000_000:
```

Comment:
bots are allowed to upload up to 8MiB, not 8MB. Besides, that limit is higher (guild.filesize_limit) if 🅱oosted.

Reply:
Also, the bot should just delete messages that it can't censor because one of the files is too big.